bundles/org.openhab.binding.electroluxappliances/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,472 @@
+	</channel-type>
+
+	<channel-type id="fanSpeed">
+		<item-type>Number</item-type>
+		<label>Fan Speed Setting</label>
+		<description>The fan speed setting.</description>
+		<state>
+			<options>
+				<option value="1">Level 1</option>
+				<option value="2">Level 2</option>
+				<option value="3">Level 3</option>
+				<option value="4">Level 4</option>
+				<option value="5">Level 5</option>
+				<option value="6">Level 6</option>
+				<option value="7">Level 7</option>
+				<option value="8">Level 8</option>
+				<option value="9">Level 9</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="workMode">
+		<item-type>String</item-type>
+		<label>Work Mode Setting</label>
+		<description>The work mode setting.</description>
+		<state>
+			<options>
+				<option value="PowerOff">Power Off</option>
+				<option value="Auto">Automatic</option>
+				<option value="Manual">Manual</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="ionizer">
+		<item-type>Switch</item-type>
+		<label>Ionizer Status</label>
+		<description>The ionizer status.</description>
+	</channel-type>
+
+	<channel-type id="uiLight">
+		<item-type>Switch</item-type>
+		<label>UI Light</label>
+		<description>The air quality light status.</description>
+	</channel-type>
+
+	<channel-type id="safetyLock">
+		<item-type>Switch</item-type>
+		<label>Safety Lock</label>
+		<description>The safety lock status.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="doorLock">
+		<item-type>Switch</item-type>
+		<label>Door Lock</label>
+		<description>The door lock status.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="doorState">
+		<item-type>Contact</item-type>
+		<label>Door Status</label>
+		<description>The door status Open/Closed.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="connectionState">
+		<item-type>Switch</item-type>
+		<label>Connection State</label>
+		<description>The connection state status.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="startTime">
+		<item-type>Number:Time</item-type>
+		<label>Start Time</label>
+		<description>The start time of a washing.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="timeToEnd">
+		<item-type>Number:Time</item-type>
+		<label>Time To End</label>
+		<description>The time to end of a program.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="applianceUiSwVersion">
+		<item-type>String</item-type>
+		<label>Appliance UI SW Version</label>
+		<description>The appliance UI SW version.</description>
+		<state readOnly="true" pattern="%s"/>
+	</channel-type>
+
+	<channel-type id="applianceTotalWorkingTime">
+		<item-type>Number:Time</item-type>
+		<label>Appliance Total Working Time</label>
+		<description>The appliance total working time.</description>
+		<state readOnly="true"/>
```

Comment:
Please provide a state pattern and i guess a `unitHint="h"`

Reply:
Fixed!

Reply:
Still don't see the unitHint, what did you change?

Reply:
This is reported in seconds:
`Channel: electroluxappliance:washing-machine:6996e7d524:33600001:appliance-total-working-time, State: 1490400`

Should I then add `unitHint="s"`?